### PR TITLE
Simple TURN for Firefox

### PR DESCRIPTION
**Pull request by trevj**
Related issue:
https://github.com/uProxy/uproxy/issues/419

[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-lib/202)

### CODE REVIEW

**Comment by trevj** [comment]
Note that this currently does not exactly work, due to the freedom-for-firefox issues related to UDP socket addresses.

File: README.md [hunk 1]
```diff
@@ -106,6 +106,11 @@ As copypaste-fredom-chat, except WebRTC traffic between the two peers is obfusca
 Simplest possible use of the TURN server (`turn-frontend` and `turn-backend`
 directories).
 
+*Note: this currently does not run on Firefox, due to these issues:*
```
Comment (bemasc) [inline README.md]: Nit: "does not work" ... I assume it does run.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ First, to install required NPMs and configure the `build/` directory for TypeScr
 ```
 
 Then, to compile the TypeScript code and build the demo apps, execute:
-```bash
+```bashs
```
Comment (bemasc) [inline README.md]: Typo?

File: README.md (hunk 1 above)
Comment (trevj) [inline README.md]: Very true!

File: README.md (hunk 2 above)
Comment (trevj) [inline README.md]: Yep, fixed!

**Comment by bemasc** [comment]
:+1:

**Pull request merged**